Kudu.Contracts/Jobs/FunctionEnvelope.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Kudu.Contracts.Infrastructure;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+
+namespace Kudu.Contracts.Jobs
+{
+    public class FunctionEnvelope : INamedObject
```

Comment:
We use the term envelope when referring to [ARM requests](https://github.com/projectkudu/kudu/blob/master/Kudu.Services/Arm/ArmUtils.cs#L15), not sure if this constitutes a naming pattern. I don't have a preference either way.